### PR TITLE
Add docs for Windows DNS configurations

### PR DESCRIPTION
**Pull request by feiskyer**
Add docs for Windows DNS configurations.

/sig windows

/cc @PatrickLang @tfogo

### CODE REVIEW

File: content/en/docs/getting-started-guides/windows/_index.md [hunk 1]
```diff
@@ -156,6 +156,33 @@ Note: this file assumes that a user previous created 'l2bridge' host networks on
 }
 ```
 
+#### DNS configurations
+
+DNS configurations for Windows containers are set by CNI plugins which support `dns` capabilities. To enable `dns` capabilities, the following options should be included in CNI configuration file:
```
Comment (neolit123) [inline content/en/docs/getting-started-guides/windows/_index.md]: `in a/the CNI`

File: content/en/docs/getting-started-guides/windows/_index.md [hunk 2]
```diff
@@ -156,6 +156,33 @@ Note: this file assumes that a user previous created 'l2bridge' host networks on
...
+
+```json
+{
+    ...
+    "capabilities": {"dns": true},
+}
+```
+
+Following DNS options from Kubelet will be passed to CNI plugins:
```
Comment (neolit123) [inline content/en/docs/getting-started-guides/windows/_index.md]: `Following` -> `The following`
`kubelet` <- lowercase

**Comment by feiskyer** [comment]
@neolit123 Thanks for reviewing. Addressed comments, PTAL.

**Comment by neolit123** [comment]
@feiskyer thanks for the update. LGTM.

**Comment by Bradamant3** [comment]
@feiskyer if this is for 1.12, please rebase against the `release-1.12` branch. Thanks!

**Comment by feiskyer** [comment]
@Bradamant3 Yep, this is for 1.12. Rebased on 1.12 branch now.

**Comment by zparnold** [comment]
@feiskyer Could you please help me find someone in sig-windows for technical review? Once they say this looks good, I'm ready to merge.

**Comment by feiskyer** [comment]
> @feiskyer Could you please help me find someone in sig-windows for technical review? Once they say this looks good, I'm ready to merge.

Sure. But the PR https://github.com/kubernetes/kubernetes/pull/67435 is still pending approve now. Let's wait a while for getting that approved.

**Comment by feiskyer** [comment]
https://github.com/kubernetes/kubernetes/pull/67435 has been moved to v1.13. Let's move this to v1.13.

/milestone v1.13

**Comment by zparnold** [comment]
For the 1.13 release
/unassign
/assign @tfogo

**Comment by feiskyer** [comment]
https://github.com/kubernetes/kubernetes/pull/67435 has been merged now.

/hold cancel

/assign @PatrickLang

**Comment by PatrickLang** [comment]
cc @daschott - this would also be good to include in the cni or Microsoft docs repos if you're doing updates there

**Comment by feiskyer** [comment]
ping @Bradamant3 for approval

**Pull request merged**